packages/kinvey-datastore2/test/operation.spec.js

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai';
+import { Operation, OperationType } from '../src';
+
+describe('OperationType', () => {
+  it('should have type Create', () => {
+    expect(OperationType.Create).to.not.be.undefined;
```

Comment:
Are we really writing unit tests for enums? I think they should at least feature the actual values - this way we are at least testing if any of them have changed unexpectedly. Having a defined value seems... inconclusive.

Reply:
This unit test should check the value of the enum rather then if is is just defined.